qa/integration-tests/src/test/java/io/camunda/it/client/SearchUserTaskTest.java

```diff
@@ -105,42 +104,26 @@ public void shouldRetrieveTaskByCandidateUser() {
 
   @Test
   public void shouldRetrieveTaskByCompletionDate() {
-    final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
-    dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
+    final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
🔧 If I'm not mistaken, you can skip using the formatter and simply do `dayBefore.toString`, for example. It will create a date string that can be parsed by our API again. For integration tests, that should be good enough.

💭 Input validation testing should happen in the REST unit tests and there you can also test with different String inputs to verify our API behaves as expected.

Reply:
I can improve that when I move the integration tests to Camunda Client 👍 
Rest API intput tests will be made on this PR (in progress) https://github.com/camunda/camunda/pull/20479

Thats my next task today.